config/config.go

```diff
@@ -143,17 +143,20 @@ func (c *Config) Ensure(fromCloud bool, logger logging.Logger) error {
 	}
 
 	for idx := 0; idx < len(c.Components); idx++ {
-		dependsOn, err := c.Components[idx].Validate(fmt.Sprintf("%s.%d", "components", idx), resource.APITypeComponentName)
+		component := c.Components[idx]
```

Comment:
Sorry for the "refactor" with the code change. Most of this is noise giving `c.Components[idx]` a name.